content/en/posts/mesa-optimizers.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Mesa Optimizers"
+date: 2023-10-22T13:25:53+02:00
+summary: Sometimes an optimization loop may create
+  agents that are themselves optimizers.
+---
+
+![Mesa Optimizers](/mesa-optimizers.jpg 'Deepl learning often involves human AI researchers tweaking some optimization process such as stochastic gradient descent, to train and output an AI model that is good at achieving some training target. Mesa optimization is the specific case where the trained model itself happens to be an optimizer, which then acts in the world and tries to move it to a certain state that\'s closer to its goal function.')
+
+# Basic
+
```

Comment:
Some easy to understand example is missing. e.g. an outer process that builds and discards robots and then each of the robots itself is able to solve how to best complete a maze.

Reply:
(as mentioned in another comment, I've added evolution + humans as an example of a mesa optimizer, and contraception as an example of misalignment for that particular case)